cyclus/pyagents.py

```diff
@@ -42,7 +42,7 @@ class Sink(Facility):
         tooltip="sink capacity",
         default=100.0,
         )
-    inventory = ts.ResourceBuffInv(capacity='max_inv_size')
+    inventory = ts.ResBufProduct()
```

Comment:
Maybe this should be
```suggestion
    inventory = ts.ResBufMaterial()
```
assuming such a thing exists

Reply:
Is a `Sink` only intended to be used with `Material`s?